internal/statemachine/classic_states.go

```diff
@@ -507,6 +507,12 @@ func (stateMachine *StateMachine) createChroot() error {
 			debootstrapCmd.String(), err.Error(), debootstrapOutput.String())
 	}
 
+	// debootstrap copies /etc/hostname from build environment; replace it
+	// with a fresh version
+	hostname := filepath.Join(stateMachine.tempDirs.chroot, "etc", "hostname")
+	hostnameFile, _ := os.OpenFile(hostname, os.O_TRUNC|os.O_WRONLY, 0644)
+	hostnameFile.WriteString("ubuntu\n")
```

Comment:
Could we have a test case that would check for this? To make sure we're indeed providing the hostname and not leaking the host's one? I don't see a test case for this.